docs/README.md

```diff
@@ -1,28 +1,116 @@
+cd unitycatalog/docs
+```
+
+### 3. Install the Required Dependencies
+
+The `requirements-docs.txt` file contains all the dependencies necessary for serving the documentation, including **MkDocs** and its plugins. First, create a virtual environment:
+
+```bash
 # Create virtual environment
 python -m venv uc_docs_venv
 
 # Activate virtual environment (Linux/macOS)
 source uc_docs_venv/bin/activate
 
 # Activate virtual environment (Windows)
-uc_docs_venv\Scripts\activate
+uc_docs_venv/Scripts/activate
 ```
 
-Install the required dependencies:
+Next, install the following dependencies into your activated virtual environment:
 
-```sh
+```bash
 pip install -r requirements-docs.txt
 ```
 
-Then serve the docs with
+### 4. Serving the Docs Locally
 
-```sh
+After installing all dependencies, you can now serve the documentation locally. Run the following command:
+
+```bash
 mkdocs serve
 ```
+
+This will start a local development server. By default, the docs will be served at `http://127.0.0.1:8000/`. You can visit this link in your browser to see your documentation live as you edit it. Any changes made to the markdown files will automatically update the browser view.
+
+### 5. Modifying the Documentation
+
+You can make changes to the documentation by editing the markdown files under the `docs` directory. MkDocs will watch for changes and automatically refresh your browser.
+
+- **Structure of the documentation** is controlled by the `mkdocs.yml` file located at the root of the repository. This file defines the site structure, theme, and various configurations.
+  
+- **Adding new pages**: Add new markdown files to the `docs` folder and then update the `mkdocs.yml` file to include the new pages in the navigation.
+
+### 6. Testing Locally
+
+After making changes to the documentation, be sure to view the changes at `http://127.0.0.1:8000/` to verify that everything looks and functions as expected.
+
+---
+
+## Deploying the Documentation
+
+After you're satisfied with the changes, please deploy the documentation to your own repo's GitHub Pages. Please include a link to this deployment for any docs PRs, as this will help streamline reviews.
+
+### 1. Ensure GitHub Pages is Enabled
+
+Make sure GitHub Pages is enabled in your repository settings. To do this, go to your repository's **Settings** and scroll down to the **GitHub Pages** section. Choose the `gh-pages` branch as the source.
+
+### 2. Deploy the Docs with MkDocs
+
+MkDocs can automatically deploy the site to the `gh-pages` branch of your repository. To do so, simply run:
+
+```bash
+mkdocs gh-deploy
+```
+
+This command will build the documentation and push the generated files to the `gh-pages` branch. Once the command is finished, your documentation will be live on GitHub Pages at:
+
+```
+https://<your-github-username>.github.io/unitycatalog
+```
+
+For example, if your GitHub username is `bobbiedraper`, your documentation will be available at:
+
+```
```

Comment:
Ditto.

Reply:
sorry i not get it, can you clarify this specific comment.

Reply:
Ah, sure. I was referring to the previous comment made on line 95:

"It would be good to have a language specified. In your case `console` should be the most appropriate one."